docs/topics/review-existing-rules.adoc

```diff
@@ -12,4 +12,20 @@ You can fork and clone the {ProductShortName} XML rules on your local machine.
 
 Rules are grouped by target platform and function. When you create a new rule, it is helpful to find a rule that is similar to the one you need and use it as a starting template.
 
-New rules are continually added, so it is a good idea to check back frequently to review the updates.
+New XML rules are occasionally added, so it is a good idea to check back frequently to review the updates.
+
+NOTE
```

Comment:
Misformatted AsciiDoc

![image](https://github.com/windup/windup-documentation/assets/106804821/c970d96d-f4d3-441c-8174-c9774b715c73)

Why are you not checking your preview?

Reply:
Fixed